jetty-core/jetty-siwe/src/main/java/org/eclipse/jetty/security/siwe/internal/SignInWithEthereumToken.java

```diff
@@ -0,0 +1,76 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.security.siwe.internal;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.function.Predicate;
+
+import org.eclipse.jetty.security.ServerAuthException;
+import org.eclipse.jetty.security.siwe.SignedMessage;
+import org.eclipse.jetty.util.IncludeExcludeSet;
+import org.eclipse.jetty.util.StringUtil;
+
+public record SignInWithEthereumToken(String scheme,
```

Comment:
Name should be a noun not a verb

Reply:
I still do not like the name. How about:
```suggestion
public record EthereumSignInToken(String scheme,
```

or just 

```suggestion
public record EthereumToken(String scheme,
```

Reply:
No because what this represents is defined the spec exactly as a `Sign-In with Ethereum` (SIWE) message, not an `EthereumSignIn` message or an `Ethereum` message.